docs/http/index.html

```diff
@@ -27,7 +27,212 @@ <h1 class="title">Module <code>spin_sdk.http</code></h1>
 <summary>
 <span>Expand source code</span>
 </summary>
-<pre><code class="python">&#34;&#34;&#34;Module with helpers for wasi http&#34;&#34;&#34;</code></pre>
+<pre><code class="python">&#34;&#34;&#34;Module with helpers for wasi http&#34;&#34;&#34;
+
+import traceback
```

Comment:
Is all this expected?!

Reply:
This is because it includes the doc changes from the deletion of the `simple` module.